### PR TITLE
python3Packages.plantuml-markdown: init at 3.6.2

### PR DESCRIPTION
**Pull request by nikstur**
###### Description of changes

New package: python3Packages.plantuml-markdown
Homepage: https://github.com/mikitex70/plantuml-markdown

The package allows to render plantuml diagrams inside markdown. It's a plugin for python3Packages.markdown

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/plantuml-markdown/default.nix [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+  propagatedBuildInputs = [
+    pkgs.plantuml
+    markdown
+    requests
+    six
+  ];
+
+  # The package uses a custom script that downloads a certain version of plantuml for testing.
+  # It also uses the unittest module so failing tests cannot be easily disabled.
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/plantuml-markdown/default.nix:36]:
```suggestion
```

File: pkgs/development/python-modules/plantuml-markdown/default.nix [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  src = fetchFromGitHub {
+    owner = "mikitex70";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-IADKU4EQHLLH5uD5iBAUiumFp5nyTNGt1LWoWdfbvJM=";
+  };
+
+  propagatedBuildInputs = [
+    pkgs.plantuml
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/plantuml-markdown/default.nix:29]:
```suggestion
    plantuml
```

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -6775,6 +6775,8 @@ in {
 
   plantuml = callPackage ../development/python-modules/plantuml { };
 
+  plantuml-markdown = callPackage ../development/python-modules/plantuml-markdown { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:6778]:
```suggestion
  plantuml-markdown = callPackage ../development/python-modules/plantuml-markdown { 
    inherit (pkgs) plantuml;
  };
```

File: pkgs/development/python-modules/plantuml-markdown/default.nix [hunk 4]
```diff
@@ -0,0 +1,68 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+  # Runtime dependencies
+, pkgs # for pkgs.plantuml
+, markdown
+, requests
+, six
+  # Test dependencies
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/plantuml-markdown/default.nix:9]:
```suggestion
, plantuml
, markdown
, requests
, six
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
@ofborg build python310Packages.plantuml-markdown

**Comment by nikstur** [comment]
@SuperSandro2000 thanks for taking the time reviewing this!

**Pull request merged**